src/shared/discover-image.c

```diff
@@ -1531,6 +1532,7 @@ int image_name_lock(const char *name, int operation, LockFile *ret) {
                 return 0;
         }
 
+        (void) mkdir_p("/run/systemd/nspawn", 0755);
         (void) mkdir_p("/run/systemd/nspawn/locks", 0700);
```

Comment:
`mkdir()` is used in the other code path to create this dir. It does not matter, but it would be good to be consistent.

Reply:
true